### PR TITLE
#3343 | FakePulseBackend: change fake backend phases to radians

### PR DESCRIPTION
**Pull request by IceKhan13**
### Summary
Change fake backends U definitions to radians + fix `u2` definition parameters
Issue https://github.com/Qiskit/qiskit-terra/issues/3343
 
### Details and comments
According to [Qiskit Backend specifications](https://arxiv.org/pdf/1809.03452.pdf) frame change phase should be in radians

### CODE REVIEW

**Comment by nkanazawa1989** [comment]
Thank you for finding this! To be more realistic, you can update the instruction sequence of mock u2 according to the conventional command defs.

```
[PulseQobjInstruction(ch='d0', name='fc', phase='-(P1)', t0=0),
 PulseQobjInstruction(ch='d0', name='Y90p_d0_3adf', t0=0),
 PulseQobjInstruction(ch='d0', name='fc', phase='-(P0)', t0=10)]
```

here `P1` comes first in the sequence.

**Comment by IceKhan13** [comment]
@nkanazawa1989 done :)

**Comment by nkanazawa1989** [comment]
LGTM

File: qiskit/test/mock/fake_openpulse_2q.py [hunk 1]
```diff
@@ -93,22 +93,22 @@ def __init__(self):
                                             samples=7*[0.j, 0.1j, 1j, 0.5 + 0j])],
             cmd_def=[Command(name='u1', qubits=[0],
                              sequence=[PulseQobjInstruction(name='fc', ch='d0',
-                                                            t0=0, phase='-P1*np.pi')]),
+                                                            t0=0, phase='-P1')]),
```
Comment (nkanazawa1989) [inline qiskit/test/mock/fake_openpulse_2q.py]: this is usually defined as `P0`. same for others.

Comment (IceKhan13) [reply qiskit/test/mock/fake_openpulse_2q.py]: fixed in https://github.com/Qiskit/qiskit-terra/pull/3785/commits/31143c4b2cc47e17047ea155c192cde98190d7f3

**Comment by IceKhan13** [comment]
@lcapelluto can you jump in? code owner review still required :)

**Pull request merged**